src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/SdkDiffExclusions.txt

```diff
@@ -40,3 +55,40 @@ msft,./sdk-manifests/x.y.z/microsoft.net.sdk.maccatalyst/*
 msft,./sdk-manifests/x.y.z/microsoft.net.sdk.macos/*
 msft,./sdk-manifests/x.y.z/microsoft.net.sdk.maui/*
 msft,./sdk-manifests/x.y.z/microsoft.net.sdk.tvos/*
+
+# linux runtimes are needed for self-contained apps - https://github.com/dotnet/source-build/issues/3507
+sb,./packs/Microsoft.AspNetCore.App.Runtime.*/*
+sb,./packs/Microsoft.NETCore.App.Runtime.*/*
+
+# netfx tooling - dumpminitool - https://github.com/dotnet/source-build/issues/3289
+msft,./sdk/x.y.z/Extensions/dump/*
+
+# netfx runtimes for dotnet-watch - https://github.com/dotnet/source-build/issues/3285
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/Microsoft.CodeAnalysis.Elfie.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/Microsoft.Win32.SystemEvents.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/runtimes/*
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/System.Configuration.ConfigurationManager.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/System.Drawing.Common.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/System.Security.Cryptography.ProtectedData.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/System.Security.Permissions.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-watch/x.y.z/tools/netx.y/any/System.Windows.Extensions.dll
+
+# netfx runtimes for dotnet-format - https://github.com/dotnet/source-build/issues/3509
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.CodeAnalysis.Elfie.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/Microsoft.Win32.SystemEvents.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/runtimes/*
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/System.Configuration.ConfigurationManager.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/System.Drawing.Common.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/System.Security.Cryptography.ProtectedData.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/System.Security.Permissions.dll
+msft,./sdk/x.y.z/DotnetTools/dotnet-format/System.Windows.Extensions.dll
+
+# netfx runtimes for containers - https://github.com/dotnet/source-build/issues/3510
+msft,./sdk/x.y.z/containerize.deps.json
+msft,./sdk/x.y.z/containerize.exe
+msft,./sdk/x.y.z/containerize.runtimeconfig.json
+msft,./sdk/x.y.z/Containers/containerize/*
```

Comment:
Thoughts on adding these to the baseline instead? These shouldn't exist in the Microsoft SDK either (see https://github.com/dotnet/source-build/issues/3510#issuecomment-1593660518).

Reply:
Yeah, that would work and match the model that's used for Intellisense files, which are in baseline, with a fix coming up for Microsoft SDK.

Reply:
Fixed with https://github.com/dotnet/installer/pull/16692/commits/b618f8cf9b904e2cac2661c30dff800c6b6e2000